src/server/snapshot/snapshotterInjected.ts

```diff
@@ -27,6 +27,8 @@ export type SnapshotData = {
   viewport: { width: number, height: number },
   url: string,
   snapshotId: string,
+  timestamp: number,
```

Comment:
This timestamp is from the page, while all  timestamps in tracing are from the node. Let's name them differently.

Also, how do we use this one?